src/env-inl.h

```diff
@@ -457,7 +462,8 @@ inline double Environment::trigger_id() {
 }
 
 inline double Environment::get_init_trigger_id() {
-  double* uid_fields = async_hooks()->uid_fields();
+  AliasedBuffer<double, v8::Float64Array> uid_fields =
+    async_hooks()->uid_fields();
```

Comment:
You could use a reference (`AliasedBuffer<double, v8::Float64Array>&`) instead, right? I think that would allow you to get rid of `AliasedBuffer`'s copy constructor (and mark that as deleted), which seems like a good idea to me, I think

Reply:
yeah, you're right.  I'll fix...

Reply:
fixed